homeassistant/components/sector_alarm/__init__.py

```diff
@@ -0,0 +1,59 @@
+"""The Sector Alarm Integration."""
+import logging
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_NAME, CONF_PASSWORD, CONF_CODE, CONF_EMAIL
+from homeassistant.helpers import discovery
```

Comment:
Imports should be grouped -> https://developers.home-assistant.io/docs/en/development_guidelines.html#ordering-of-imports